src/resources/faqs.yml

```diff
@@ -1,15 +1,17 @@
+# Moment doesn't give us a way to get the ordinal of a date (the "st"/"nd"/"rd"/"th" part),
+# but they do give us a way to get the day and the ordinal e.g. "11th"/"22nd".
+# The string splitting of `graduationDate.format('Do')` is a hack to extract the ordinal for the date
+# so we can put it in superscript.
+
+# <% graduationDate = dates.getEarliestGraduationDateToAccept() %>
+# <% formattedGraduationDate = graduationDate.format('D') + '<sup>' + graduationDate.format('Do').split(graduationDate.format('D'))[1] + '</sup> of ' + graduationDate.format('MMMM YYYY') %>
```

Comment:
This is working but I was wondering how, since it looks like it's in a YAML comment

Reply:
This is because EJS is parsing the raw file, before it gets parsed as YAML, so the comments are still there. It seemed a bit neater (since it's not really part of the YAML) though it is potentially confusing. I've ended up needing to change this anyway in the webpack branch.